shared/gh/partials/admin-batch-edit.html

```diff
@@ -89,7 +94,10 @@
                                     <th>
```

Comment:
Per comment earlier, it would be good to avoid rendering the checkbox `th` and `td` elements when we don't need it to avoid the extra space in front of the title.